src/main/scala/uk/gov/hmrc/play/frontend/bootstrap/FrontendGlobal.scala

```diff
@@ -81,7 +81,7 @@ abstract class DefaultFrontendGlobal
     super.onStart(app)
   }
 
-  def filters = if (enableSecurityHeaderFilter) frontendFilters ++ Seq(securityFilter) else frontendFilters
+  def filters = if (enableSecurityHeaderFilter) Seq(securityFilter) ++ frontendFilters  else frontendFilters
 
```

Comment:
We tried adding a test for the DefaultFrontendGlobal. There is quite a bit of setup required for the other filters i..e., MetricsFilter etc for testing. Hence, we added a test just to verify the affected behaviour. Please let us know if there is a simpler option to test this.